components/03-modules/quantity-update/_quantity-update.scss

```diff
@@ -0,0 +1,89 @@
+// Input
+$quantity-update__input-border                       : 1px solid $gray-dark !default;
+$quantity-update__input-width                        : 48px !default;
+$quantity-update__input-color                        : $gray-dark !default;
+
+// Button
+$quantity-update__button-background                  : $white !default;
+$quantity-update__button-border                      : 1px solid $gray-dark !default;
+$quantity-update__button-border-color--disabled      : $gray !default;
+$quantity-update__button-background-hover--disabled  : $white !default;
+$quantity-update__button-border-color-hover--disabled: $white !default;
+$quantity-update__button-fill-hover--disabled        : $gray !default;
+
+// Icon
+$quantity-updater__icon-fill                         : $gray-dark !default;
+$quantity-updater__icon-spacing                      : 12px !default;
+
+.quantity-update {
+    display: flex;
+
+    &__input {
+        border-top: $quantity-update__input-border;
+        border-bottom: $quantity-update__input-border;
+        border-left: 0;
+        border-right: 0;
+        width: $quantity-update__input-width;
+        height: $quantity-update__input-width;
+        color: $quantity-update__input-color;
+        text-align: center;
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    &__button {
+        border: 1px solid $gray-dark;
+        background: $white;
```

Comment:
I guess you should use some variables here...maybe those defined on line 7 & 8 :> ?

Reply:
thanks! 😀